### PR TITLE
[IMP] hr_recruitment : Improve the UX of recruitment app

### PR DESCRIPTION
**Pull request by jugj-odoo**
task - 3175879

### CODE REVIEW

File: addons/hr/views/hr_job_views.xml [hunk 1]
```diff
@@ -19,15 +19,14 @@
                             <page string="Recruitment" name="recruitment_page">
                                 <group>
                                     <group name="recruitment">
-                                        <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
```
Comment (sofiagvaladze) [inline addons/hr/views/hr_job_views.xml:22]: As we discussed, we can't just remove the field from the view in stable, cause that might break customization. Instead what we could do is add `invisible="1"` to it :bulb: 

Applies to this and all the other places that we remove field from the view

File: addons/hr_recruitment/models/hr_recruitment.py [hunk 2]
```diff
@@ -148,7 +148,7 @@ class Applicant(models.Model):
     date_closed = fields.Datetime("Hire Date", compute='_compute_date_closed', store=True, readonly=False, tracking=True)
     date_open = fields.Datetime("Assigned", readonly=True)
     date_last_stage_update = fields.Datetime("Last Stage Update", index=True, default=fields.Datetime.now)
-    priority = fields.Selection(AVAILABLE_PRIORITIES, "Appreciation", default='0')
+    priority = fields.Selection(AVAILABLE_PRIORITIES, "Evaluation", default='0')
```
Comment (sofiagvaladze) [inline addons/hr_recruitment/models/hr_recruitment.py:151]: .pot here and at other places as well where we add or change the text :innocent:

File: addons/hr_recruitment/views/hr_recruitment_views.xml [hunk 3]
```diff
@@ -491,7 +492,7 @@
         <field name="model">hr.job</field>
         <field name="priority">200</field>
         <field name="arch" type="xml">
-            <form string="Create a Job Position" class="o_hr_job_simple_form" >
```
Comment (sofiagvaladze) [inline addons/hr_recruitment/views/hr_recruitment_views.xml:494]: Unnecessary change, especially in stable :innocent:

**Review by sofiagvaladze — CHANGES_REQUESTED** [review]
@jugj-odoo good job  :raised_hands: 

As discussed here are some point, as we are targeting the stable, we need to take extra care of some changes. 

Thanks!

**Comment by sofiagvaladze** [comment]
@jugj-odoo thanks for the changes :raised_hands: 

Didn't notice on the first go, final comment from me - the commit messages (community and enterprise) does not reflect all the modules that are affected by the commit, could you change it? :innocent: 

Thanks!

**Review by kba-odoo — CHANGES_REQUESTED** [review]

**Comment by kba-odoo** [comment]
robodoo r+

**Comment by kba-odoo** [comment]
robodoo r+

**Pull request closed without merging**